includes/Core/Authentication/Clients/Google_Proxy_Client.php

```diff
@@ -37,7 +37,15 @@ final class Google_Proxy_Client extends Google_Client {
 	 * @since 1.1.2
 	 * @var string
 	 */
-	protected $proxy_base_path = '';
+	protected $proxy_base_path = Google_Proxy::BASE_URL;
+
+	/**
+	 * Callback to pass a potential exception to while refreshing an access token.
+	 *
+	 * @since n.e.x.t
+	 * @var callable|null
+	 */
+	protected $token_exception_callback = null;
```

Comment:
Stylistic preference: all declared properties are null by default, so setting that here is redundant 😄 
```suggestion
	protected $token_exception_callback;
```